packages/arb-avm-cpp/data_storage/src/arbcore.cpp

```diff
@@ -1139,6 +1210,26 @@ uint256_t ArbCore::maxCheckpointGas() {
     }
 }
 
+std::variant<rocksdb::Status, MachineStateKeys> ArbCore::getMaxCheckpoint(
```

Comment:
Maybe just return the variant and then use the function we have to get a `MachineOutput` out of the variant since both member types have one

Reply:
:+1: